src/Http/Http.Results/src/ObjectResult.cs

```diff
@@ -37,6 +37,11 @@ public ObjectResult(object? value, int? statusCode)
         /// </summary>
         public int? StatusCode { get; set; }
 
+        /// <summary>
+        /// Gets the value for the <c>Content-Type</c> header.
+        /// </summary>
+        public string? ContentType { get; init; }
```

Comment:
Why is this `init;` but everything else is `set;`

Reply:
No reason. I'll switch it to make it consistent
```suggestion
        public string? ContentType { get; set; }
```